### ISSUE TITLE
Actions are failing.

### ISSUE THREAD

**Issue by ktsakalozos** [juju-solutions/layer-apache-kafka#3]
Actions check the availability of the service (service kafka status) but we switched to starting the service from a thread.

https://github.com/juju-solutions/layer-apache-kafka/commit/e24b95a6320689b75cb2793286ea8ae314c7a92b#diff-01385d1c5b298f3e82a82645dcd54af4R124

**Comment by merlijn-sebrechts** [comment]
Why did you switch to starting a service from a thread? I changed it to the upstart service because I need Kafka to automatically start when the host starts and I need Kafka to automatically restart when it crashes.

**Comment by ktsakalozos** [comment]
Hi Merlijn,

We did not change the service to start as a thread, we did the opposite. With this commit: https://github.com/juju-solutions/layer-apache-kafka/commit/f49cb227101339e660f85643a99490317a2fcc63 Kafka service starts as a system service either in systemd or upstart depending on what is available. There are two templates  https://github.com/juju-solutions/layer-apache-kafka/tree/master/templates do they match your needs?

(At some point we were indeed thinking of starting the service as a thread because we had to support both upstart and systemd, but in the end we ended up having the two templates)

The issue reported here was that the actions where checking the service status and is addressed by starting Kafka a a service. I will close the issue as soon as we finish this discussion.
